### ISSUE TITLE
About modifying the assets of lostcity' s .jar file in 1.12.2.

### ISSUE THREAD

**Issue by ysrwhs** [McJty/LostCities#544]
I have modified the "lostcitychest.json" in mods/lostcities-1.12-2.0.22.jar and it comes out that all chests(in building) become empty. I 've tried this in several ways including covering the defaul .json with the modified one in winrar, or decompress the .jar file then repack it into .rar after modify it. It all seems that the mod itself is working as normol but just all the chests in buildings do not contain anying. I believe this question has been asked before, I apologize for interrupting you by asking the same question again if this is the case. And aslo I will keep looking for identical issues may has been discussed to find the answers. 
I used a command imported by "LootTweaker" to track the loot_table of any unopened chest while in game, and it says that the loot table has become {"pools": []} (which is empty)which corresspond to containing nothing. But if open the .jar file with winrar, the lostcitychest.json file is still in the correct path and contains the very loot table I wrote. I think the way I modify the .jar file may somehow be wrong.
I uploaded my modified loot table below. I think it should be grammatically correct. I changed it into a txt file to match the type filter.
[lostcitychest.txt](https://github.com/McJtyMods/LostCities/files/11179544/lostcitychest.txt)

**Comment by McJty** [comment]
Don't modify the jar! That's not how you provide custom assets. Lost Cities has a system for changing the assets. You can find more information here:

https://www.mcjty.eu/docs/mods/lost-cities/

**Comment by ysrwhs** [comment]
I have found why I did not managed to modify the lostcitychest loot table, and it 's because I 've changed the size of that particular json by adding (only one) additional entry(lines) to it. If I keep the amount of entries in that file untouched by only modifying the strings and integers in it, the modified loot table can work as I desired. The assets system you provided seems to me that it is capable of customzing the loot table together with items of additional mods only in version later than 1.18, 
or it maybe because I really didn' t make it through in understanding how to use "conditions". I just didn 't understand how to add a brand new loot table to my minecraft so that the $lostcities/userassets.json can successfully locate and use it in my custom "conditions". Any way thanks for your help and I' m glad it can be solved in an easier way.
![无标题](https://user-images.githubusercontent.com/130160509/230700349-c3cd21d7-6086-4279-a358-ea60856704d8.png)
